content/agent/_index.md

```diff
@@ -64,15 +64,15 @@ By default the Agent binds 3 ports on Linux and 4 on Windows and OSX:
 |------|---------------------------------------------------------------------------------------------|
 | 5000 | Exposes runtime metrics about the Agent.                                                    |
 | 5001 | Used by the Agent CLI and GUI to send commands and pull information from the running Agent. |
-| 5002 | Used by the GUI server on Windows and OSX.                                                  |
+| 5002 | Serves the GUI server on Windows and OSX.                                                   |
 | 8125 | Used for the DogStatsD server to receive external metrics.                                  |
 
 ### The Collector
 The collector gathers all standard metrics every 15 seconds. Agent v6 embed a Python2.7 interpreter to run integrations and [custom checks][1].
 
 ### The Forwarder
 
-The Agent forwarder listens for incoming requests over HTTP to send metrics over HTTPS to Datadog. Buffering prevents network splits from affecting metric reporting. Metrics are buffered in memory until a limit in size or number of outstanding send requests are reached. Afterwards the oldest metrics are discarded to keep the forwarder's memory footprint manageable. Logs are sent over an SSL-encrypted TCP connection to Datadog.
+The Agent forwarder takes payloads from the collector and sends them to Datadog.
```

Comment:
could we use a wording similar to the google doc we wrote ?

```
The agent forwarder sends metrics over HTTPS to Datadog. Buffering allows for network splits not to affect metric reporting. Metrics will be buffered in memory until a limit in size or number of outstanding requests to send is reached. Afterwards the oldest metrics will be discarded to keep the agent's memory footprint manageable. Logs are sent over an SSL-encrypted TCP connection to Datadog HQ.
```

What do you think ?